auxiliary/rasterstotiles/rasterstotiles.py

```diff
@@ -1,12 +1,12 @@
-import sys
 import subprocess
-from osgeo import gdal,ogr,osr
+from osgeo import gdal, osr
 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
 
-#Windows
-GDAL2TILE_PATH = 'C:\Python27\Lib\site-packages\osgeo\scripts\gdal2tiles.py'
-#Linux
-#GDAL2TILE_PATH = '/usr/local/bin/gdal2tiles.py'
+# Windows
+# GDAL2TILE_PATH = 'C:\Python27\Lib\site-packages\osgeo\scripts\gdal2tiles.py'
+# Linux
+GDAL2TILE_PATH = '/usr/local/bin/gdal2tiles.py'

```

Comment:
I'm a little uncomfortable with this change. I guess it's not generic enough either way.

Reply:
Yeah, I think there's a generic cross platform way that might be worth looking into. For Linux, I would expect users to already have the tool in their $PATH and it shouldn't need to be specified. Not sure how Windows handles things.

Reply:
Go ahead and merge anyway.

Reply:
Agreed. We need to find a better way to address multiplatform Python programs that reference GDAL, but let's merge it anyways since it doesn't conflict with other programs. Also, it's a simple fix to change the GDAL2TILE_PATH variable.